backend/urls.py

```diff
@@ -18,8 +18,17 @@
 from django.urls import path, include
 from django.conf.urls.static import static
 from django.conf import settings
+from pulse.views.question_views import getAllQuestions
+
+
 
 urlpatterns = [
     # path('admin/', admin.site.urls),
+    path('api/getAllQuestions/', lambda request: get_all_questions_view(request), name='get_all_questions'),
```

Comment:
lines 27 - 28 should not be included (we are already pulling in all of the URLs from `pulse.urls` in line 29